source/includes/investor-payments.md.erb

```diff
@@ -647,3 +647,103 @@ Extract a fee from a specified investor's account cash balance
 | bankAccountId    | string | The ID of the bank account details the fees are being sent to                 |
 | reference        | string | The payment reference to be used for the fee transaction                      |
 | status           | string | The status of the fee transfer process. Possible values are: <br>`PENDING`<br>`CLEARED`<br>|
+
+## `GET /reports/investor/cash/balance`
+
+```http
+
+GET /reports/investor/cash/balance?asOf=2020-12-25T15:00:00Z&id={investorId1}&id={investorId2}&id={investorId3} HTTP/1.1
+Host: api-sandbox.goji.investments
+Content-Type: application/json
+Authorization: Basic ...
+
+
+
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
+    "investorId": "INVESTOR-ID-1",
+    "gia": {
+      "totalBalance": { "amount": 10000.0, "currency": "GBP" },
+      "uninvestedCashBalance": { "amount": 5000.0, "currency": "GBP" },
+      "pendingIncomingTransferBalance": { "amount": 0, "currency": "GBP" },
+      "queuedForInvestmentBalance": { "amount": 5000.0, "currency": "GBP" },
+      "investedBalance": { "amount": 0, "currency": "GBP" }
+    },
+    "isa": {
+      "totalBalance": { "amount": 1000.0, "currency": "GBP" },
+      "uninvestedCashBalance": { "amount": 0.0, "currency": "GBP" },
+      "pendingIncomingTransferBalance": { "amount": 0, "currency": "GBP" },
+      "queuedForInvestmentBalance": { "amount": 1000.0, "currency": "GBP" },
+      "investedBalance": { "amount": 0, "currency": "GBP" }
+    },
+    "sipp": {
```

Comment:
Is it worth removing this since it may change with upcoming sipp work?

Reply:
I have no idea what direction the work is going in, so would need to take a cue from those that do...

Reply:
I thought about this some more this morning and think just leave it for now.